src/Entity/PhoneNumberSettingsInterface.php

```diff
@@ -85,33 +85,33 @@ public function getVerificationLifetime();
    * @return $this
    *   Return phone number settings for chaining.
    */
-  public function setVerificationLifetime($lifetime);
+  public function setVerificationCodeLifetime($lifetime);
 
   /**
-   * Determine if phone number field values are removed when phone number
+   * Determines if phone number field values are removed when phone number
    * verifications expire.
    *
    * @return bool
    *   Whether to remove phone number field values
    */
-  public function isVerificationPhoneNumberPurge();
+  public function canPurgeVerificationPhoneNumber();
```

Comment:
Another convention in Drupal. My site directory has >40x occurrences of is than can.